src/test/java/org/isf/medicalstock/test/Tests.java

```diff
@@ -1194,6 +1195,16 @@ public void testMovementGetOrigin() throws Exception {
 		assertThat(movement).isNotNull();
 		assertThat(movement.getSupplier()).isEqualTo(movement.getOrigin());
 	}
+	
+	@Test
+	public void testDeleteLastMovement() throws Exception {
+		int code = setupTestMovement(false);
+		Optional<Movement> movement = movementIoOperationRepository.findById(code); 
+		assertThat(movement).isPresent();
+		movBrowserManager.deleteLastMovement(movement.get());
+		Optional<Movement> movement2 = movementIoOperationRepository.findById(code); 
+		assertThat(movement2).isNotPresent();
+	}
```

Comment:
I think we should test also the cases when the deletion is denied according with `deleteLastMovement()` checks

Reply:
okay